intake/container/serializer.py

```diff
@@ -102,10 +91,28 @@ def decode(self, bytestr, container):
             self._compressor.decompress(bytestr), container)
 
 
+try:
+    import snappy
+
+
+    class SnappyCompressor(object):
+        name = 'snappy'
+
+        def compress(self, data):
+            return snappy.compress(data)
+
+        def decompress(self, data):
+            return snappy.decompress(data)
+
+
+    compressors = [SnappyCompressor(), GzipCompressor(),
```

Comment:
A somewhat minor thing, but it would avoid repetition to set `compressors = [GzipCompressor(), NoneCompressor()]` once, above, and then _append_ to the `compressors` here.